cmake/InstallRules.cmake

```diff
@@ -25,6 +25,7 @@ install(
   ARCHIVE #
   COMPONENT ${package_name}_development
   INCLUDES #
+  FILE_SET CXX_MODULES
```

Comment:
I'm not familiar with this option but I'll trust this does something reasonable :)